### PR TITLE
Update DG and UG

### PR DESCRIPTION
**Pull request by tohyuting**
Write-ups are not completed yet, will finish it in the next few commits.

### CODE REVIEW

File: src/main/java/seedu/clinic/logic/commands/HelpCommand.java [hunk 1]
```diff
@@ -100,7 +99,7 @@ private String generateHelpGenericMessage() {
         String aboutListCommand = "list\nYou can list all suppliers and warehouses by using list command.";
         String aboutViewCommand = "view\nYou can view a specific warehouse or supplier using"
                 + " the view command.";
-        String aboutUpdateCommand = "update\nYou can update stock of a product in the warehouse by"
+        String aboutUpdateCommand = "update\nYou can update a product in a supplier or warehouse by"
```
Comment (jeffreytjs) [inline src/main/java/seedu/clinic/logic/commands/HelpCommand.java]: "product in a supplier" sounds a bit weird, maybe "a product managed by a supplier or warehouse" is better

File: src/main/java/seedu/clinic/logic/commands/HelpCommand.java [hunk 2]
```diff
@@ -111,37 +110,27 @@ private String generateHelpGenericMessage() {
     }
 
     private String generateHelpAddMessage() {
-        String generalDescriptionOfAdd = "There are three types of add command to add warehouse,"
-                + " add supplier or add a product to a supplier.";
+        String generalDescriptionOfAdd = "Add\nAdd command allows you to add warehouse or supplier"
```
Comment (jeffreytjs) [inline src/main/java/seedu/clinic/logic/commands/HelpCommand.java:113]: Add warehouse or supplier to a supplier?

File: src/test/java/seedu/clinic/logic/commands/HelpCommandTest.java [hunk 3]
```diff
@@ -22,12 +22,11 @@ public void execute_genericHelp_success() {
                 + " each command, feel free to copy our website User Guide URL,"
                 + " which can be found by pressing F1.";
 
-        String aboutAddCommand = "add\nYou can add warehouse, supplier or product to a supplier using the add"
-                + " command";
+        String aboutAddCommand = "add\nYou can add warehouse, supplier using the add command";
```
Comment (jeffreytjs) [inline src/test/java/seedu/clinic/logic/commands/HelpCommandTest.java]: Should be add "warehouse or supplier" instead

File: src/test/java/seedu/clinic/logic/commands/HelpCommandTest.java [hunk 4]
```diff
@@ -36,7 +35,7 @@ public void execute_genericHelp_success() {
         String aboutListCommand = "list\nYou can list all suppliers and warehouses by using list command.";
         String aboutViewCommand = "view\nYou can view a specific warehouse or supplier using"
                 + " the view command.";
-        String aboutUpdateCommand = "update\nYou can update stock of a product in the warehouse by"
+        String aboutUpdateCommand = "update\nYou can update a product in a supplier or warehouse by"
```
Comment (jeffreytjs) [inline src/test/java/seedu/clinic/logic/commands/HelpCommandTest.java]: Same as above, "product managed by a supplier or warehouse" would be better

File: src/test/java/seedu/clinic/logic/commands/HelpCommandTest.java [hunk 5]
```diff
@@ -51,37 +50,28 @@ public void execute_genericHelp_success() {
 
     @Test
     public void execute_addHelp_success() {
-        String generalDescriptionOfAdd = "There are three types of add command to add warehouse,"
-                + " add supplier or add a product to a supplier.";
+        String generalDescriptionOfAdd = "Add\nAdd command allows you to add warehouse or supplier"
```
Comment (jeffreytjs) [inline src/test/java/seedu/clinic/logic/commands/HelpCommandTest.java:53]: Here too, we should review it if these are retrieved from our individual commands

File: docs/UserGuide.md [hunk 6]
```diff
@@ -200,17 +208,17 @@ Format: `list`
...
-* The supplier/warehouse NAME specified is case-insensitive.
+* The TYPE specified should be one of these values: s or w representing supplier or warehouse.
+* The supplier/warehouse INDEX specified should be within range of the supplier/warehouse list, corresponding to the number on the list.
 
 Examples:
 
-* `view supplier supplierA` Displays all the information associated with supplierA e.g. address, contact, email, products sold by the supplier etc.
-* `view warehouse warehouseB` Displays all the information associated with warehouseB e.g. address, all the products stored in the warehouse etc.
+* `view ct/s 1` Displays all the information associated with supplier at index 1 in the supplier list e.g. name, contact, email, remarks, products sold by the supplier.
+* `view ct/w 2` Displays all the information associated with at index 2 in the warehouse list e.g. name, contact, address, remarks, all the products stored in the warehouse.
```
Comment (zhengweii) [inline docs/UserGuide.md]: Missing warehouse!

**Review by zhengweii — APPROVED** [review]
LGTM

**Pull request merged**